src/app/actions.ts

```diff
@@ -62,3 +63,28 @@ const speciesFuseIndex = new Fuse<Species>(speciesData as readonly Species[], {
 export async function findSpecies(query: string): Promise<SpeciesGetResponse> {
   return speciesFuseIndex.search(query).slice(0, 10);
 }
+
+export async function getNearbySightings(longitude: number, latitude: number) {
+  if (!process.env.EBIRD_API_TOKEN) {
+    throw new Error("Missing eBird API token");
+  }
+
+  const url = `${EBIRD_BASE_API_URL}/data/obs/geo/recent?lat=${latitude}&lng=${longitude}`;
+
+  const searchParams = new URLSearchParams({
```

Comment:
This isn't used so for now can be removed.

Reply:
Are you gonna remove it in this PR? Or is that gonne be in a follow up?

Alternatively, instead of removing it we can use it to set `lat` and `lng` in a slightly more idiomatic way than using the template string literal.

e.g

```ts
const searchParams = new URLSearchParams({
  lat: latitude,
  lng: longitude
});
```

Then down in the fetch you can do something like:

```ts
fetch(`${url}?${searchParams.toString()}`)
```

The reason this is a preferred pattern is because it makes it easier to apply optional parameters if we ever get there. E.g something like the `dist` param for nearby sightings. You can do something like:

```ts
if(dist) {
  searchParams.append('dist', dist);
}
```

Then it automatically gets applied to the URL when it exists.

Reply:
There's some type work to do to put in the searchParams as you've described. I think a separate PR will work a treat for that.

Reply:
Alright ace, let's open up another PR and we can talk types and implementation!